.github/CONTRIBUTING.md

```diff
@@ -43,6 +41,17 @@ After writing content, [submit your work](https://github.com/magento/merchdocs/w
 
 **Tip!** If you are not sure where to start contributing, search for issues with the `help wanted` and `good first issue` labels. These are researched and ready to work!
 
+## User Guide versions
+
+Our repository currently publishes two versions of the Magento User Guide: 2.3.x and 2.4.x (current).
```

Comment:
If a contribution is applicable to both versions, a PR must be created to 2.4 (master).

Reply:
I am going to change this slightly, as `port` is probably not the best term to use. But we will take changes to 2.3-production for now, until the 2.3.6 release in October. After that, we will close down external contributions for the 2.3 guide and only do them with prior approval internally. I do not want to force people to makes change in master if they are working on 2.3. Plus it is cleaner to go forward rather than backward when it comes to documentation.

Reply:
My first impression was that 2.3 contributions will be mostly restricted to rare editorial and minor updates.

Reply:
After 2.3.6, that will be the case.